src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -56,91 +60,106 @@
+        assertParseSuccess(parser, TYPE_DESC_IRENE + NRIC_DESC_IRENE + NAME_DESC_IRENE + PHONE_DESC_IRENE
+                + AGE_DESC_IRENE + PRIORITY_DESC_BOB + PRIORITY_DESC_IRENE, new AddCommand(expectedPatient));
+
+        // multiple ages - last age accepted
+        assertParseSuccess(parser, TYPE_DESC_IRENE + NRIC_DESC_IRENE + NAME_DESC_IRENE + PHONE_DESC_IRENE
+                + AGE_DESC_BOB + AGE_DESC_IRENE + PRIORITY_DESC_IRENE, new AddCommand(expectedPatient));
+
+    }
+
+    @Test
+    public void parse_allFieldsPresentDonor_success() {
         //Testing all fields present for a Donor
         Donor expectedDonor = JOHN;
 
         // whitespace only preamble
         assertParseSuccess(parser, PREAMBLE_WHITESPACE + TYPE_DESC_JOHN + NRIC_DESC_JOHN
-            + NAME_DESC_JOHN + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
+                + NAME_DESC_JOHN + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
 
         // multiple names - last name accepted
         assertParseSuccess(parser, TYPE_DESC_JOHN + NRIC_DESC_JOHN + NAME_DESC_AMY
-            + NAME_DESC_JOHN + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
+                + NAME_DESC_JOHN + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
 
         // multiple phones - last phone accepted
         assertParseSuccess(parser, TYPE_DESC_JOHN + NRIC_DESC_JOHN + NAME_DESC_JOHN
-            + PHONE_DESC_AMY + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
+                + PHONE_DESC_AMY + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
 
         // multiple nrics - last nric accepted
         assertParseSuccess(parser, TYPE_DESC_JOHN + NRIC_DESC_AMY + NRIC_DESC_JOHN + NAME_DESC_JOHN
-            + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
+                + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
 
         // multiple types - last type accepted
         assertParseSuccess(parser, TYPE_DESC_BOB + TYPE_DESC_JOHN + NRIC_DESC_JOHN
-            + NAME_DESC_JOHN + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
+                + NAME_DESC_JOHN + PHONE_DESC_JOHN + AGE_DESC_JOHN, new AddCommand(expectedDonor));
 
+        // multiple ages - last age accepted
+        assertParseSuccess(parser, TYPE_DESC_JOHN + NRIC_DESC_JOHN
+                + NAME_DESC_JOHN + PHONE_DESC_JOHN + AGE_DESC_BOB + AGE_DESC_JOHN, new AddCommand(expectedDonor));
     }
 
-    //    @Test
-    //    public void parse_optionalFieldsMissing_success() {
-    //        // zero tags
-    //        Person expectedPerson = new PersonBuilder(AMY).build();
-    //        assertParseSuccess(parser, NAME_DESC_AMY + PHONE_DESC_AMY,
-    //                new AddCommand(expectedPerson));
-    //    }
+    @Test
+    public void parse_allFieldsPresentDoctor_success() {
+        Person expectedDoctor = new PersonBuilder(AMY).build();
```

Comment:
now that doctor class has been implemented, shall we change this to reflect as such?

Reply:
Changed